recipes/term-image/meta.yaml

```diff
@@ -0,0 +1,46 @@
+{% set name = "term-image" %}
+{% set version = "0.1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/term-image-{{ version }}.tar.gz
+  sha256: 4e4cbf8d6fb670f20cbf5907aaf5e1c38eebed45be62682a83c170beaed83a5f
+
+build:
+  entry_points:
+    - term-img=term_img.__main__:main
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - pillow >=8.3,<10.0
+    - python >=3.6
+    - requests >=2.23,<3.0
+    - urwid >=2.1,<3.0
+
+test:
+  imports:
+    - term_img
+  commands:
+    - pip check
+    - term-img --help
+  requires:
+    - pip
+
+about:
+  home: https://github.com/AnonymouX47/term-img
+  summary: Display images in the terminal
+  license: MIT
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - MattChanTK
```

Comment:
Please comment from this account, showing that they/you are a willing maintainer.

Reply:
my bad somehow git always put that wrong username in, not sure why